Source/Core/DiscIO/VolumeCreator.cpp

```diff
@@ -123,16 +96,15 @@ void VolumeKeyForPartition(IBlobReader& _rReader, u64 offset, u8* VolumeKey)
 	memset(IV, 0, 16);
 	_rReader.Read(offset + 0x44c, 8, IV);
 
-	bool usingKoreanKey = false;
 	// Issue: 6813
 	// Magic value is at partition's offset + 0x1f1 (1byte)
 	// If encrypted with the Korean key, the magic value would be 1
 	// Otherwise it is zero
-	if (Reader.Read8(0x3) == 'K' && Reader.Read8(offset + 0x1f1) == 1)
-		usingKoreanKey = true;
+	u8 using_korean_key = 0;
+	Reader.Read8(offset + 0x1f1, &using_korean_key);
```

Comment:
I have reasons to believe that the 'K' check shouldn't be there, but I suppose that's best to do in another PR. I'll change this so that the logic is the same as before.